public/app/plugins/datasource/loki/querybuilder/operations.ts

```diff
@@ -324,7 +324,7 @@ Example: \`\`error_level=\`level\` \`\`
       orderRank: LokiOperationOrder.LineFilters,
       renderer: getLineFilterRenderer('|~'),
       addOperationHandler: addLokiOperation,
-      explainHandler: (op) => `Return log lines that match regex \`${op.params[0]}\`.`,
+      explainHandler: (op) => `Return log lines that match a \`Go language\` regex pattern. \`${op.params[0]}\`.`,
```

Comment:
I think we should come up with something better than `Go language`. The RegExp syntax is not really tied to go, but is used there a lot.
Could we just name it `RE2 syntax` or something along the lines of that?

https://github.com/google/re2/wiki/Syntax

Reply:
of course, thats my bad i didn't know that. i'll change it now

Reply:
It was our fault. We specified it like that in the original issue. Thanks both!